sample-bundle/porter.yaml

```diff
@@ -0,0 +1,87 @@
+# This is the configuration for Porter
+# You must define steps for each action, but the rest is optional
+# See https://getporter.org/bundle/manifest/file-format/ for a description of all the allowed fields in this document.
+# See https://getporter.org/author-bundles for documentation on how to configure your bundle.
+# Uncomment out the sections below to take full advantage of what Porter can do!
+
+# Optional indicator of the resource type of this file
+schemaType: Bundle
+
+# Version of the porter.yaml schema used by this file.
+schemaVersion: 1.0.1
+
+# Name of the bundle
+name: nomad-example-bundle
+
+# Version of the bundle. Change this each time you modify a published bundle.
+version: 0.1.0
+
+# Description of the bundle and what it does.
+description: "An example bundle that uses Nomad"
+
+# Registry where the bundle is published to by default
+registry: "localhost:5000"
+
+# Declare and optionally configure the mixins used by the bundle
+mixins:
+  - exec:
+  - nomad:
+      NOMAD_ADDR: "http://IP_ADDRESS_HERE" # don't forget the http://
```

Comment:
Hmm, awkward that these are capitalized but the other ones aren't. Could we match them all to have the same typing (snake case, camel case, w/e)

Reply:
I did this intentionally because they act exactly like the corresponding environment variables nomad users are used to, so i named them like the environment variables as well.

Reply:
That totally makes sense, I feel like it could just feel unnatural due to how we're dealing with parameters with the rest of the bundle. 
But we should probably show how you can pass in `env vars` into a param like [how we do there](https://porter.sh/wiring/#parameters)

Reply:
Specifically this part:
```
- name: database-name
  type: string
  default: mydb
  env: DATABASE_NAME
```